### ISSUE TITLE
Where are the "phrases.json" "protein_sequence_mapping.json" and embedding model and vectore store?

### ISSUE THREAD

**Issue by YifanDengWHU** [sornkL/ProDVa#2]
Hi, I tried to run ProDVA. I downloaded the checkpoints from huggingface but didn't find phrases.json file for protein_fragment_mapping_file, I tried "phrase_tokenizer/vocab.json" but it didn't work.
Also, for the embedding model and vector store and "protein_sequence_mapping.json", I couldn't find them either.

Thank you in advance!

**Comment by sornkL** [comment]
Hi, thank you for your questions!

`phrases.json` is model-agnostic and it depends on the datasets you use. For instance, if you want to evaluate on the Mol-Instructions test set, the `phrases.json` file can be found [here](https://huggingface.co/datasets/nwliu/Molinst-SwissProtCLAP/blob/main/phrases.json). Note that the file serves as a mapping file (protein–fragment pairs), and you may create your own mapping file depending on your own dataset.

Similarly, the vector store and the protein_sequence mapping file (instruction-protein pairs) are model-agnostic. The file should correspond to the vector store you use, which in our case is the training file. For instance, the protein_sequence mapping file for the Mol-Instructions can be found [here](https://huggingface.co/datasets/nwliu/Molinst-SwissProtCLAP/blob/main/training.json). And the related vector store can be found [here](https://huggingface.co/datasets/nwliu/Molinst-SwissProtCLAP/tree/main/index). You may also create your own training set and build the FAISS index accordingly.

The embedding model used in our paper is [NeuML/pubmedbert-base-embeddings](https://huggingface.co/NeuML/pubmedbert-base-embeddings). Please refer to #1 for more details!
